tests/bsim/README.md

```diff
@@ -0,0 +1,72 @@
+This folder contains tests meant to be run with BabbleSim's physical layer
+simulation, and therefore cannot be run directly from twister.
+
+The compile.sh and run_parallel.sh scripts are used by the CI system to build
+the needed images and execute these tests in batch.
+
+You can also run them manually if desired, but be sure to call them setting
+the variables they expect. For example, from Zephyr's root folder, you can run:
+
+```
+WORK_DIR=${ZEPHYR_BASE}/bsim_out tests/bsim/compile.sh
+RESULTS_FILE=${ZEPHYR_BASE}/myresults.xml SEARCH_PATH=tests/bsim tests/bsim/run_parallel.sh
+```
+
+Or to run only a specific subset, e.g. host advertising tests:
+
+```
+WORK_DIR=${ZEPHYR_BASE}/bsim_out tests/bsim/bluetooth/host/compile.sh
+RESULTS_FILE=${ZEPHYR_BASE}/myresults.xml \
+SEARCH_PATH=tests/bsim/bluetooth/host/adv tests/bsim/run_parallel.sh
+```
+
+Check the run_parallel.sh help for more options and examples on how to use this script to run these
+tests in batch.
+
+After building the tests' required binaries you can run a test directly with its individual
+test scripts.
+
+For example you can build the required binaries for the networking tests with
+
+```
+WORK_DIR=${ZEPHYR_BASE}/bsim_out tests/bsim/net/compile.sh
+```
+and then directly run one of the tests:
+
+```
+tests/bsim/net/sockets/echo_test/tests_scripts/echo_test_802154.sh
+```
+
+
+Conventions about these tests scripts
+-------------------------------------
+
+Each test is defined by a shell script with the extension `.sh`.
+Scripts starting with an underscore (`_`) are ignored.
+
+Each of these test scripts expects the binaries their require already built, and will execute
+the required simulated devices and physical layer simulation with the required command line
+options.
+
+Tests must return 0 to the invoking shell if the test passes, and not 0 if the test
+fails.
+
+It is recommended to have a single test for each test script.
+
+Each test must have a unique simulation id, to enable running different tests in parallel.
```

Comment:
Should we add a reference to `SIMULATION_ID`?

Reply:
I'm not sure I follow. We don't define anywhere in the zephyr docs a SIMULATION_ID (beware this is a disconnected README file, not linked from the docs)

Reply:
True, and I guess the `SIMULATION_ID` in the scripts are only locally used. Nvm :)